docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,115 +1,135 @@
+The contents of `my_project` should now look like this:
 
 [,bash]
 ----
-pip install cairo-nile
-nile init
-----
-
-=== Install the library
+$ ls
 
-[,bash]
+Scarb.toml src
 ----
-pip install openzeppelin-cairo-contracts
-----
-
-WARNING: Installing directly through GitHub may contain incomplete or breaking implementations.
-While we aim not to introduce such changes, we still strongly recommend installing through https://github.com/OpenZeppelin/cairo-contracts/releases/[official releases].
 
-=== Use a basic preset
+==== Install the library
 
-Presets are ready-to-use contracts that you can deploy right away.
-They also serve as examples of how to use library modules.
-xref:extensibility.adoc#presets[Read more about presets].
+Add the library to your project be adding the openzeppelin dependency to the `Scarb.toml` file:
 
-[,cairo]
+[,text]
 ----
-// contracts/MyToken.cairo
-
-%lang starknet
-
-from openzeppelin.token.erc20.presets.ERC20 import (
-    constructor,
-    name,
-    symbol,
-    totalSupply,
-    decimals,
-    balanceOf,
-    allowance,
-    transfer,
-    transferFrom,
-    approve,
-    increaseAllowance,
-    decreaseAllowance
-)
+[dependencies]
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0" }
 ----
 
-Compile and deploy it right away:
+WARNING: Make sure the correct tag matching the release is selected.
 
-[,bash]
-----
-nile compile
-
-nile deploy MyToken <name> <symbol> <decimals> <initial_supply> <recipient> --alias my_token
-----
-
-NOTE: `<initial_supply>` is expected to be two integers i.e.
-`1` `0`.
-See xref:utilities.adoc#uint256[uint256] for more information.
+=== Basic usage
 
-=== Write a custom contract using library modules
+Copy the following account contract code into `src/lib.cairo`, and run `scarb build` to compile it:
 
-xref:extensibility.adoc#libraries[Read more about libraries].
-
-[,cairo]
+[,javascript]
 ----
-%lang starknet
-
-from starkware.cairo.common.cairo_builtins import HashBuiltin
-from starkware.cairo.common.uint256 import Uint256
-from openzeppelin.security.pausable.library import Pausable
-from openzeppelin.token.erc20.library import ERC20
-
-(...)
-
-@external
-func transfer{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
-    recipient: felt, amount: Uint256
-) -> (success: felt) {
-    Pausable.assert_not_paused();
-    return ERC20.transfer(recipient, amount);
+#[starknet::contract]
+mod MyAccount {
+    use openzeppelin::account::Account;
+    use openzeppelin::account::account::PublicKeyTrait;
+    use openzeppelin::account::interface;
+    use openzeppelin::introspection::interface::ISRC5;
+    use openzeppelin::introspection::src5::SRC5;
+    use starknet::account::Call;
+
+    #[storage]
+    struct Storage {}
```

Comment:
maybe we could add a comment stating the storage is being defined and handled by each used module?